lib/meetup_bot/workers/meetup_cache_worker.ex

```diff
@@ -5,16 +5,16 @@ defmodule MeetupBot.MeetupCacheWorker do
 
   alias MeetupBot.MeetupCache
   alias MeetupBot.Meetup
-  alias MeetupBot.GDG
+  alias MeetupBot.Constants
   alias OpenTelemetry.Tracer
 
   @impl true
   def perform(%Oban.Job{}) do
     Tracer.with_span "oban.perform" do
       Tracer.set_attributes([{:worker, "MeetupCacheWorker"}])
 
-      (Meetup.fetch_upcoming_meetups() ++ GDG.fetch_live_events())
-      |> MeetupCache.update()
+      meetup_events = Meetup.fetch_upcoming_meetups()
+      MeetupCache.sync_upcoming_external_events(meetup_events, Constants.meetup_source())
 
       :ok
     end
```

Comment:
If we end up not splitting this in multiple worker this would be

```elixir
  def perform(%Oban.Job{}) do
    Tracer.with_span "oban.perform" do
      Tracer.set_attributes([{:worker, "MeetupCacheWorker"}])

      events = (Meetup.fetch_upcoming_meetups() ++ GDG.fetch_live_events())

      MeetupCache.upsert(events)
      MeetupCache.delete_upcoming_not_present_in_external_source(events)

      :ok
    end
```